src/block/count-up/edit.js

```diff
@@ -60,7 +62,21 @@ const Edit = props => {
 		<Fragment>
 
 			<InspectorTabs />
+			<InspectorBlockControls>
+				<PanelAdvancedSettings
+					title={ __( 'Speed', i18n ) }
+					id="count-up">
+					<AdvancedRangeControl
+						label={ __( 'Duration', i18n ) }
+						attribute="duration"
+						min={ 1 }
+						max={ 10 }
```

Comment:
People might want more control over the duration, please use `ms` for the value of the duraction. min `100`, max `10000` and step `100`